cmd/clair-load-test/reports.go

```diff
@@ -155,7 +160,7 @@ func getContainersList(ctx context.Context, testRepoPrefix string, hitSize int,
 func reportAction(c *cli.Context) error {
 	ctx := c.Context
 	conf := NewConfig(c)
-	if (c.String("containers") == "" && conf.TestRepoPrefix == "") || ((c.String("containers") != "") && conf.TestRepoPrefix != "") {
+	if (c.String("containers") == "" && c.String("testrepoprefix") == "") || ((c.String("containers") != "") && c.String("testrepoprefix") != "") {
 		return fmt.Errorf("Please specify either of --containers or --testrepoprefix options. Both are mutually exclusive")
 	}
 	validLayers := []int{-1, 5, 10, 15, 20, 25, 30, 35, 40}
```

Comment:
This can be a package level variable, it can also be checked in the flag's [action](https://cli.urfave.org/v2/examples/flags/#flag-actions) without having to get this far in the code.

Reply:
Added a `Before` function to validate this check in the `cli.Command` itself.

Reply:
+1 on using the before function, I was actually referring to the `validLayers := []int{-1, 5, 10, 15, 20, 25, 30, 35, 40}` variable, it might be nice to have it at the package level (as it's essentially static), then check the passed value in the flag's action. Just a thought